lib/restful_resource_bugsnag/version.rb

```diff
@@ -1,3 +1,3 @@
 module RestfulResourceBugsnag
-  VERSION = "0.5.2"
+  VERSION = "0.5.3"
```

Comment:
NOTE: I just released it manually, via `rake publish` and using credetials in Bitwarden.
Couldn't be bothered to sort out this CI automation for it right now, since we publish this once in a blue moon.